cli/flox/doc/flox-pull.md

```diff
@@ -7,53 +7,72 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-pull -
+flox-pull - pull environment from FloxHub
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] pull [ `<options>` ] [ \--force ] [ ( -m | \--main) ]
+```
+flox [ <general-options> ] pull
+     [-d=<path>]
+     [-a]
+     [-r=<owner/name> | <owner/name> | [-f]]
+```
 
 # DESCRIPTION
 
-(`git`) Push or pull metadata to the environment's `floxmeta` repository,
-and in the `pull` case also proceed to render the environment.
-With this mechanism environments can be pushed and pulled between machines
-and within teams just as you would any project managed with `git`.
-
-With the `--force` argument flox will forceably overwrite either the
-upstream or local copy of the environment based on having invoked
-`push` or `pull`, respectively.
-
-With the `(-m|\--main)` argument `flox (push|pull)` will operate on the
-"floxmain" branch, pulling user metadata from the upstream repository.
-Cannot be used in conjunction with the `-e|\--environment` flag.
-
-With the `--no-render` argument `flox pull` will fetch and incorporate
-the latest metadata from upstream but will not actually render or create
-links to environments in the store. (Flox internal use only.)
+Pull an environment from FloxHub and create a local reference to it,
+or, if an environment has already been pulled, retrieve any updates.
+
+When pulling an environment for the first time, `-d` specifies the directory in
+which to create that environment.
+The remote environment is specified in the form `<owner/name>`,
+and it may optionally be preceded by `-r`.
+
+When updating an environment that has already been pulled, `-d` specifies which
+environment to update.
```

Comment:
Suggestion: maybe also say that you don't need `-d` if you're referring to the current directory/repo that you're in.

Reply:
```
If `-d` is not specified and the current directory contains an environment, that
environment is synced.
```

I think we detect upwards from the current directory and use the last active environment as well - not sure if we need to explain that somewhere